src/content/protocols/ethena/usde.md

```diff
@@ -0,0 +1,329 @@
+
+The `StakedUSDeV2` contract implements a cooldown period, _Stakers_ must wait 7 days to complete a withdrawal. After the cooldown the `USDe` can be withdrawn by calling `withdraw` which transfers `USDe` from the `USDeSilo` contract to the _Staker_. The cooldown period is configurable by calling `setCooldownDuration` and is capped at 90 days max.
+
+The `StakedUSDeV2` contract also includes a blacklist mechanism managed by addresses holding the `BLACKLIST_MANAGER_ROLE`. If a user is blacklisted, they are prevented from staking or unstaking `USDe`. This could result in _loss of funds_ if users are unable to retrieve their staked assets and and if they have to sell `sUSDe` on the open market at a discount. It can lead to a complete _loss of funds_ if their `sUSDe` is burned from their wallet and redistributed to other _Stakers_ via the `redistributeLockedAmount` function, which is controlled by the `DEFAULT_ADMIN_ROLE`.
+
+Yield from the various strategies is transferred into the system via the `StakingRewardsDistributor` contract. The `StakingRewardsDistributor` contract holds `USDe` rewards and regularly calls `transferInRewards` to transfer them into `StakedUSDeV2`. The `StakingRewardsDistributor` is operated by an address labeled [Operator](#security-council), which is itself appointed by the [Dev Multisig](#security-council). While the [Operator](#security-council) can delay transfers, they cannot redirect funds or change the recipient. This minimizes the risk of _loss of unclaimed yield_, though misconfiguration or inaction may still delay yield distribution. The [Dev Multisig](#security-council) can replace the [Operator](#security-council) to prevent _loss of unclaimed yield_ at any time.
+
+The ability to rescue tokens from the vault, handled by `rescueTokens`, is restricted to addresses with the `DEFAULT_ADMIN_ROLE`. Although this function cannot extract `USDe`, it can affect other tokens accidentally sent to the vault.
+
+# Dependencies
+
+`USDe` is only slightly overcollateralized and relies on various strategies to maintain stability and peg to the US-Dollar. The strategies rely on various counterparties to function correctly. If any of the counterparties fail, the solvency of the protocol is at risk. Ethena has a [Reserve Fund](#security-council) to provide a buffer in case of negative performance of the diversified yield strategy. Ethena employs diversification within each strategy by relying on various counterparties (various CEXs, different Custodians, diverse centralized Stablecoins and LSTs), as well as diversify in different strategies (short futures, staking yield, long spot).
```

Comment:
In the second paragraph, I would just say that you list those below instead of them again